flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/CreatingExecutionGraph.java

```diff
@@ -79,6 +81,11 @@ private void handleExecutionGraphCreation(
                     throwable);
             context.goToFinished(context.getArchivedExecutionGraph(JobStatus.FAILED, throwable));
         } else {
+            for (ExecutionVertex vertex :
+                    executionGraphWithVertexParallelism.executionGraph.getAllExecutionVertices()) {
```

Comment:
This seems to be in line with the default scheduler 👍. If I understand that correctly scheduled basically means, I know where these vertices will be executed, but I haven't told the TMs yet (which is covered by the DEPLOYED state), is that correct?

Reply:
It means that the scheduler now actively prepares the deplyoment of these executions, by requesting resources and assigning slots.